types/universal-router/generateUrls.d.ts

```diff
@@ -0,0 +1,14 @@
+import UniversalRouter, { Params } from ".";
+
+export interface generateUrlsOptions {
```

Comment:
As far as I am familiar with common TS style, interface should be named in Pascal Case. You could name it `GenerateUrlOptions` or `UrlGeneratorOptions` or maybe just `Options` since this is a separate file and interface won't directly conflict with UniversalRouter `Options`.

Reply:
IMO that doesn't matter as long as there is no rule for that, but you are right!